tests/TransitiveTypeFieldExporterTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Crell\Serde;
+
+use Crell\Serde\Attributes\Field;
+use Crell\Serde\Formatter\ArrayFormatter;
+use Crell\Serde\PropertyHandler\Exporter;
+use Crell\Serde\PropertyHandler\Importer;
+use Crell\Serde\Records\Transitive;
+use Crell\Serde\Records\TransitiveExample;
+use Crell\Serde\Records\Transitives;
+use Crell\Serde\Records\TransitiveTypeField;
+use PHPUnit\Framework\Attributes\DataProvider;
+use PHPUnit\Framework\Attributes\Test;
+use PHPUnit\Framework\TestCase;
+
+class TransitiveTypeFieldExporterTest extends TestCase
+{
+    #[Test]
+    public function can_export_the_right_values(): void
+    {
+        $exporter = new class () implements Exporter {
+            public function exportValue(Serializer $serializer, Field $field, mixed $value, mixed $runningValue): mixed
+            {
+                return $serializer->formatter->serializeString($runningValue, $field, $value->name);
+            }
+
+            public function canExport(Field $field, mixed $value, string $format): bool
+            {
+                return $field->typeField instanceof TransitiveTypeField;
+            }
+        };
+        
+        $serde = new SerdeCommon(handlers: [$exporter], formatters: [new ArrayFormatter()]);
```

Comment:
Don't provide a literal formatter yourself.  See how the other tests handle the formatter.  It's setup that way so that the same test bank can work on all array-based tests (and any symmetric formatter, really), so we get 3 tests for one ever time. :-)  Please also include the validator step as the other tests do.